orchestration/orchestrate_unified.py

```
@@ -0,0 +1,411 @@
+        """Check system dependencies and report status."""
+        dependencies = {"tmux": "tmux", "git": "git", "gh": "gh", "claude": "claude"}
+
+        missing = []
+        for name, command in dependencies.items():
+            try:
+                result = subprocess.run(
+                    ["which", command], check=False, capture_output=True, text=True
+                )
+                if result.returncode != 0:
+                    missing.append(name)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)